src/Limiters/limiter.jl

```diff
@@ -0,0 +1,121 @@
+# function sign_preserving_limiter!(
+#     ρstar::Fields.Field,
+#     space::Spaces.SpectralElementSpace2D,
+# )
+
+# stub for sign_preserving_limiter
```

Comment:
For now I left a stub here with no implementation of the sign_preserving_limiter, as per Oswald suggestion